mirror-main.go

```diff
@@ -330,15 +331,9 @@ func doMirrorSession(session *sessionV6) {
 		json.Unmarshal([]byte(urlScanner.Text()), &sURLs)
 		// Verify if previously copied, notify progress bar.
 		if isCopied(sURLs.SourceContent.URL.String()) {
-			statusCh <- doMirrorFake(sURLs, progressReader)
```

Comment:
done,

by the way,  progressReader.ProgressBar.Add64(sURLs.SourceContent.Size) is fast, it takes 10 seconds for 100 millions entries in my little old system (i5, from 2011)